backend/src/main/kotlin/de/neuefische/backend/user/User.kt

```diff
@@ -0,0 +1,18 @@
+package de.neuefische.backend.user
+
+import de.neuefische.backend.common.Medium
+import org.bson.BsonObjectId
+import org.bson.codecs.pojo.annotations.BsonId
+import org.springframework.data.mongodb.core.index.Indexed
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document("user")
+data class User(
+    @BsonId
+    val id: BsonObjectId = BsonObjectId(),
```

Comment:
Does the Id work as an object in Mongo? I had crazy issues and had to use a String id... so I was just curious 😝

Reply:
Looks like it, if you want to use the database ID directly on your model/class. You can also add a unique id of your own that has nothing to di with it (actually I did something of the sort with the name).